README.md

```diff
@@ -27,7 +28,13 @@ If you wish to use [fms-acceleration](https://github.com/foundation-model-stack/
 ```
 pip install git+https://github.com/foundation-model-stack/fms-acceleration.git#subdirectory=plugins/framework
 ```
-`fms-acceleration` is a collection of plugins that packages that accelerate fine-tuning / training of large models, as part of the `fms-hf-tuning` suite. For more details on see [this section below](#fms-acceleration).
+`fms-acceleration` is a collection of plugins that packages that accelerate fine-tuning / training of large models, as part of the `fms-hf-tuning` suite. For more details see [this section below](#fms-acceleration).
+
+### 2. Build from source 
+
+We have committed a `poetry.lock` file to allow reproducible enviornments. If building from source, you can clone the repository and use poetry to install as mentioned in [development docs](/CONTRIBUTING.md#development) 
+
+If building in a dockerfile use `poetry export --format requirements.txt` can install same dependencies from the lock file. Maintainers regularly update the lock file. 
```

Comment:
this line is not very clear.. this will do not any installation correct? Its just an export of the locked deps to a requirements file?

Reply:
ya its just to export the requirements and same lock file and then we have to install. hmm should be just link our dockerfile as example?

Reply:
> @VassilisVassiliadis there is a PR here to add back the [.fms-accel] dep. It was removed before with the intention of being added back https://github.com/foundation-model-stack/fms-hf-tuning/pull/223

@fabianlim I'll rebase the PR once #223 makes its way into the `main` branch. 

> why will poetry constraint packages under optional dependencies? that makes no sense

I'm actually not sure. I assume that they do this so that `pip install` and `pip install -E fms-accel` both end up installing the same non-optional dependencies (in our case  transformers).